### PR TITLE
Fix travis (and local) builds

### LINKED ISSUE #207: Book cannot be built anymore
The Pillar Pharo 4.0 CI build being used for building this book is not available anymore:. The dead URL is:

```
https://ci.inria.fr/pharo-contribution/job/Pillar/PHARO=40,          VERSION=stable,VM=vm/lastSuccessfulBuild/artifact/Pillar.zip
```

It also seems that newer pillar version don't support the `inputFiles` configuration in `pillar.conf` anymore so just using an newer version doesn't work.

### PR DESCRIPTION
**Pull request by macie**
This branch fixes #207 by:
- updating _pillar_ download scripts (to most recent)
- updating _travis_ config to run automatic builds.

What is not working (yet):
- compiling book - something is wrong when I try to compile book. How did you make it before?

### CODE REVIEW

**Comment by Ducasse** [comment]
Hi 
Thanks for this contribution. Now I do not know what you are doing. Because this book should be migrated to use the same configuration and version than for example the SmaccBooklet or the LearningObjectOriented book.

**Comment by macie** [comment]
My primary motivation was to read about Pharo on my e-book reader (#209). To do that, I wanted to compile book on my computer (to one-file HTML document, and than convert to EPUB). But it was impossible.

So I updated _pillar_ download script and (by the way) simplify _travis_ configuration. But now, after reading your configuration examples I get stuck. Documentation on compilation is unclear. I feel overwhelmed by this procedure (_travis_ with _jenkins_ with _docker_ with _makefiles_ - probably created by package _Pillar-MakefileGeneration_ - with _bintray_) - it's very complicated to create a readable book.

I'd like to help with this project, but unfortunately I miss direction. It would be great, if you can provide me some guidelines.

**Comment by Ducasse** [comment]
If it works for you then use it. 
Now travis is simple this is just to compile each time someone without latex installation does a commit. 
Docker is for people that do not have latex installed. 
So if you have latex installed you do not need docker. Docker is for the travis configuration. 
I do not get why you need to touch the travis configuration because this is for the validation on travis and you do not care about it. 
Now I do not have the time to use the latest version of Pillar (pillar 50) the one that it is used in the booklet for example or the learning OOP book.
Basically we should copy the support folder and the makefile and update the pillar.conf but I'm super super busy right now because it may not work the first time.

**Comment by Ducasse** [comment]
We moved to the pillar 7 dev branch (soon to be released). 
Thanks again for your time. We will check the Epub support because we have it in the booklet on physche or a reflective core.

**Comment by iNode** [comment]
> We moved to the pillar 7 dev branch (soon to be released).
> Thanks again for your time. We will check the Epub support because we have it in the booklet on physche or a reflective core.

Any progress with actual fix? Build still seems to be broken.

**Comment by Ducasse** [comment]
Hi 

I do not have brain ressources for this issue now. I will try to have a look at it after mid of july. 

S.
